portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeStandardHead.vm

```diff
@@ -58,18 +45,17 @@
                     "format" : [#foreach( $format in $mathJaxFormat )"${format}",#end]
                 },
                 toolTitles : {#foreach ($entry in $toolTitles.entrySet())"$entry.key": "$entry.value",#end},
-                notifications: { pushEnabled: $!notificationsPushEnabled, debug: $!debugNotifications },
-                serviceName: "$serviceName",
             };
         </script>
 
         <script type="module" src="/webcomponents/bundles/base.js$!{portalCDNQuery}"></script>
 
         #parse("/vm/morpheus/snippets/title-snippet.vm")
 
-        ## TRINITY-43 Review before merging to master
         #parse("/vm/morpheus/snippets/portalCSS-snippet.vm")
 
+        <link rel="manifest" href="/manifest.json" />
```

Comment:
Hi @adrianfish why was line added? This file doesn't exist and giving a console error right now.

Reply:
It went in by accident :) I'll remove it in one of my other current PRs. Thanks for spotting it.